easybuild/easyconfigs/s/scGSVA/scGSVA-0.0.14-foss-2022a-R-4.2.1.eb

```
@@ -0,0 +1,54 @@
+easyblock = 'Bundle'
+
+name = 'scGSVA'
+version = '0.0.14'
+versionsuffix = '-R-%(rver)s'
+
+homepage = 'https://github.com/guokai8/scGSVA'
+description = """
+scGSVA provides wrap functions to do GSVA analysis for single cell data. And scGSVA 
+includes functions to build annotation for almost all species. scGSVA also provides 
+function to generate figures based on the GSVA results.
+"""
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+source_urls = [
+    'https://cran.r-project.org/src/contrib/Archive/%(name)s',
+    'https://cran.r-project.org/src/contrib/',
+    'https://cran.freestatistics.org/src/contrib',
+]
+
+dependencies = [
+    ('R', '4.2.1'),
+    ('R-bundle-Bioconductor', '3.15', versionsuffix),
+]
+
+exts_defaultclass = 'RPackage'
+exts_default_options = {
+    'sources': ['%(name)s_%(version)s.tar.gz'],
+    'source_urls': source_urls,
+}
+
+exts_list = [
+    ('babelgene', '22.3', {
+        'checksums': ['0f4a69bd472ce67682cb03e8186238bd3484fb9bbf185c6f291baa2d3481659c'],
+    }),
+    ('msigdbr', '7.5.1', {
+        'checksums': ['dc30487bdf3594425ae9faec1ca0d7d0cd7278f4f177689133f92880e74acaca'],
+    }),
+    (name, version, {
+        'source_urls': ['https://github.com/guokai8/scGSVA/archive/refs/tags/'],
+        'sources': ['%(version)s.tar.gz'],
+        'checksums': ['64a0795cdb6fcc0e8088f3327b2ec178b8610d097c38b16e7ed1efab7c839151'],
+    }),
+]
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': ['scGSVA'],
```

```suggestion
    'dirs': [name],
```
It's a minor point, but a little bit cleaner.